meorg_client/docs/cli.md

```diff
@@ -0,0 +1,144 @@
+# Command-Line Usage
+
+The client can be used over the command line in a standard UNIX environment.
+
+> NOTE: The command-line utilities will only work if you have been granted API access by modelevaluation.org.
+
+## Set up credentials.
+
+Credentials to the system are stored in the user home directory `$HOME/.meorg/credentials.json`. To set up credentials follow these steps:
+
+1. Get an account on modelevaluation.org. Take note of the email and password used.
+2. Run the command `meorg initialise` on the machine where you have the client installed.
+3. Follow the prompts.
+
+The system will attempt to authenticate with modelevaluation.org, this will write the credentials file upon success. 
+
+> NOTE: This will overwrite any existing credentials.json
+
+Alternatively, you can create a credentials file at the target filepath manually in the following format:
+
+```json
+{
+    "email": "user@example.com",
+    "password": "SuperSecretPassword"
+}
+```
+
+Once credentials are set up, you may use the command-line utilities listed alphabetically below. However, given the asynchronous nature of the server requests, a typical workflow is more useful.
+
+## Typical Workflow
+
+A typical workflow interacting with the server is as follows:
+
+1. Set up your credentials as above.
+2. Take note of the `$MODEL_OUTPUT_ID` by visting the appropriate page on modelevaluation.org.
```

Comment:
What is the appropriate page on modelevaluation.org? Can you attach screen shots?

Reply:
At the moment it is a bit frustrating to get the model output id, as we are working on getting it displayed prominently on the details page.

Sadly, we have to describe a "go here, click here, copy this bit" situation until this is resolved.

I've added an example for how to get this information to the docs.